pymatgen/io/vasp/sets.py

```diff
@@ -1215,41 +1215,47 @@ def from_prev_calc(cls, prev_calc_dir, **kwargs):
         return input_set.override_from_prev_calc(prev_calc_dir=prev_calc_dir)
 
 
-class MatPESStaticSet(MPStaticSet):
+class MatPESStaticSet(DictSet):
```

Comment:
Inheriting from `DictSet` means `MatPESStaticSet.from_prev_calc` is no longer defined so the tests need to change.

Reply:
The tests are still failing. You can run them using

```py
pytest tests/io/vasp/test_sets.py -k TestMatPESStaticSet -vv
```

Requires `pip install pytest` if you haven't installed it yet. Should print sth like

```txt
___________ TestMatPESStaticSet.test_init ___________

self = <tests.io.vasp.test_sets.TestMatPESStaticSet testMethod=test_init>

    def test_init(self):
        # check if default INCAR settings are loaded
        default = MPStaticSet(self.struct)
        incar = default.incar
        # assert incar["GGA"] == "Pe"
        assert incar["ALGO"] == "Normal"
>       assert incar["EDIFF"] == 1.0e-05
E       assert 0.0012000000000000001 == 1e-05

tests/io/vasp/test_sets.py:747
```